src/named.rs

```diff
@@ -227,6 +227,9 @@ pub enum NamedChain {
 
     #[cfg_attr(feature = "serde", serde(alias = "kakarot-sepolia"))]
     KakarotSepolia = 1802203764,
+
+    #[cfg_attr(feature = "serde", serde(alias = "kakarot-etherlink-testnet"))]
```

Comment:
Is kakarot here right?

Reply:
Not at all… :sweat: sorry about that, and good catch, thanks!